hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieMultiTableDeltaStreamer.java

```diff
@@ -355,6 +414,12 @@ public static void main(String[] args) throws IOException {
     @Parameter(names = {"--checkpoint"}, description = "Resume Delta Streamer from this checkpoint.")
     public String checkpoint = null;
 
+    /**
+     * Fetch data from multi sources.
+     */
+    @Parameter(names = {"--fetch-from-multi-sources"}, description = "Fetch data from multi sources and ingest to single sink table")
+    public Boolean isFetchFromMultiSources = false;
```

Comment:
ditto, update the variable name and description

Reply:
ok, I've modified it.